src/ILCompiler.Compiler/src/Compiler/UserDefinedTypeDescriptor.cs

```diff
@@ -180,6 +180,10 @@ private uint GetVariableTypeIndex(TypeDesc type, bool needsCompleteIndex)
                     variableTypeIndex = GetEnumTypeIndex(type);
 
                     GetTypeIndex(type, false); // Ensure regular structure record created
```

Comment:
Why should we create record for enum structure? Seems like we doesn't use it.

Reply:
@davidwrighton This came with https://github.com/dotnet/corert/pull/3932 . Could you please take a look?